test/testcafe/spec/EnrollAuthenticatorDataPhoneView_spec.js

```diff
@@ -59,12 +64,12 @@ test.requestHooks(mock)('voice mode click and extension will get shown', async t
   await t.expect(enrollPhonePage.getSaveButtonLabel()).eql('Receive a code via voice call');
 
   // Extension field is shown
-  await t.expect(enrollPhonePage.extensionIsHidden()).eql(false);
-  const extensionText = await enrollPhonePage.getElement('.phone-authenticator-enroll__phone-ext').innerText;
+  await t.expect(await enrollPhonePage.extensionIsHidden()).eql(false);
+  const extensionText = await enrollPhonePage.extensionText();
   await t.expect(extensionText.trim()).eql('Extension');
```

Comment:
Instead of these where we get the text then assert that it is what we expect, I think we could make is simpler by just trying to get the field by the label/text and if it isnt there it will fail. 

We have [this ](https://github.com/okta/okta-signin-widget/blob/0.1/test/testcafe/framework/page-objects/components/BaseFormObject.js#L34)function in the baseformObject. Then you can use it here like:
```
await t.expect(await enrollPhonePage.formFieldExistsByLabel('Extension')).eql(true);
```